CHANGELOG.md

```diff
@@ -0,0 +1,238 @@
+# Changelog
+
+## [Unreleased]
```

Comment:
@vweevers Will you add `v3.0.0-rc1`? I figured it's best if you do it since you have everything fresh in your head.

Reply:
https://github.com/Level/level.js/issues/108